src/main/java/com/launchacademy/reviews/services/ReviewService.java

```diff
@@ -0,0 +1,26 @@
+package com.launchacademy.reviews.services;
+
+import com.launchacademy.reviews.models.Review;
+import com.launchacademy.reviews.repositories.ReviewRepository;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewService {
+  private ReviewRepository reviewRepository;
+
+  @Autowired
+  public ReviewService(ReviewRepository reviewRepository) {
+    this.reviewRepository = reviewRepository;
+  }
+
+  public List<Review> findAllReviews(Pageable pageable){
+    return (List<Review>) reviewRepository.findAll(pageable);
+  }
+
+  public Review save(Review review){
+    return reviewRepository.save(review);

```

Comment:
Ideally this would have been split into two smaller PR's. The first focused on creating Reviews and having them display on a films show page, while the second PR would work on adding a form to allow a visitor to the site to create a review

Reply:
Noted